### DISCUSSION TITLE
Extract String instruments

### DISCUSSION THREAD

**Discussion by rarnett** [Anjok07/ultimatevocalremovergui#1208] [General]
Can anyone recommend a UVR5 process or specific model that will extract string instruments similar to how VR's 17_HP-Wind_Inst-UVR removes wind instruments?

**Comment by ghost** [comment]
Unfortunatly, there's currently no public "strings" model to my knowledge....

**Reply by kipper65** [reply]
On sites like mvsep.com - Lalalai etc there are possibilities to extract the Strings stem.. This means that it is possible to obtain this result.. but how???

See >>> https://mvsep.com/algorithms/58?lang=en

Best Regards
